docs/migrate-to-vector-tiles-1.0.md

```diff
@@ -0,0 +1,246 @@
+# Migrate to vector tiles 1.0 from earlier releases
+
+If you are connecting to the Mapzen vector tile service using a URL that starts with `https://vector.mapzen.com`, you need to consider your migration path for Mapzen vector tiles 1.0 and take action.
+
+The 1.0 release of the Mapzen vector tiles service has some [major changes and enhancements](https://mapzen.com/blog/v1-vector-tile-service/), and Mapzen recommends all existing users of the service migrate to it. This process is not automatic, and does require you to modify your project.
+
+At a minimum, you need to update your code or stylesheets, such as [Tangram YAML files](https://mapzen.com/documentation/tangram/sources/), to point to the service's new URL. Any additional work to use the 1.0 release depends on how much customization and styling you are using.
+
+## Update the service URL
+
+You can determine which release of the service you are using by looking at the URL in your code. If you see `vector.mapzen.com`, you are using the deprecated tile service.
+
+To use vector tiles 1.0, replace your URL with the following scheme, depending on which tile format you want:
+
+- GeoJSON: `http://tile.mapzen.com/mapzen/vector/v1/all/{z}/{x}/{y}.json`
+- TopoJSON: `http://tile.mapzen.com/mapzen/vector/v1/all/{z}/{x}/{y}.topojson`
+- Mapbox Vector Tile: `http://tile.mapzen.com/mapzen/vector/v1/all/{z}/{x}/{y}.mvt`
+
+You need to append your own [API key](https://mapzen.com/documentation/overview/) at the end for access to the full rate limits.
+
+Mapzen offers several different types of tiles in vector and raster formats and the service combines data from multiple sources. Because of this, it was not possible to redirect existing queries automatically to use the 1.0 URL.
+
+## Choose whether to use some or all layers
+
+A vector tiles request can be for one or more individual layers, such as `roads` or `landuse`, or for `all` layers in the service.
+
+With the 1.0 release, you may find benefits to requesting `all` layers instead of custom ones. You should run benchmark tests for both custom and `all` layers on your target devices and network connections. If you find that one method is clearly faster than the other, then choose it. If the results are about the same, then use `all` because it has a simpler service architecture.
+
+If you choose `all`, you can use client-side cartography to limit the display of the layers. For example, if you are using Tangram to draw your map, you can use data [filters](https://mapzen.com/documentation/tangram/Filters-Overview/).
+
+Both custom and `all` layers are part of Mapzen's global tile server cache. This is beneficial because tiles from the cache do not count toward your rate limits and can be served faster, yet the map data in the tiles can still be updated regularly.
```

Comment:
> @rmarianski to confirm (and make sure we're not double counting against API limits when the custom request is on-demand).

:white_check_mark: That's right.
Specifically, the xonacatl request itself is not rate limited, and it's just the subrequest to the `all` layer that may require a new tile to be generated. This may change in the future if we find that xonacatl itself needs to be guarded (and here we would need to avoid double counting). Anyway, :+1: that the rate limits info has been removed, because it might just add confusion, which I probably just did.